extensions/admin-tool-ext-win/package.json

```diff
@@ -10,7 +10,7 @@
 	"aiKey": "AIF-5574968e-856d-40d2-af67-c89a14e76412",
 	"engines": {
 		"vscode": "^1.30.1",
-		"azdata": "*"
+		"azdata": "^1.8.0"
```

Comment:
I would recommend to use >=1.8.0

^ means same major version and great or equal to the minor and build numbers, e.g. 2.0.0 of ADS will be evaluated to false.

Reply:
https://devhints.io/semver

Reply:
this is sematic version standard

Reply:
Good point, thanks. 

There's a couple other extensions with this issue as well, I'll put out a fix for them in a sec.